tests.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think we should gitignore this file. I don't know why it is not in the template created by github...

Reply:
I'll do this on my branch